tests/functional/test_eos.py

```diff
@@ -0,0 +1,263 @@
+import json
+from pathlib import Path
+
+from ragger.backend import SpeculosBackend
+from ragger.backend.interface import RAPDU, RaisePolicy
+from ragger.navigator import NavInsID, NavIns
+from ragger.utils import pack_derivation_path
+
+from apps.eos import EosClient, ErrorType, STATUS_OK
+from apps.eos_transaction_builder import Transaction
+
+
+TESTS_ROOT_DIR = Path(__file__).parent
+
+# Proposed EOS derivation paths for tests ###
+EOS_PATH = pack_derivation_path("m/44'/194'/12345'")
+
+speculos_expected_public_key = "04a478ace4ac9cdbc8ccfe5560940a2c"
+speculos_expected_public_key += "cdc96d4f7789e7dd4074dbe1eb7865b0"
+speculos_expected_public_key += "0889833972fdafcbd25e71f7515c27c1"
+speculos_expected_public_key += "23449309873e0d16fea13abd2697c035ef"
+
+speculos_expected_address = "EOS85fjM4VLKEYZwJE5FBUhXR3HaFno1t7fpukBfzjm9xUHgzLpuV"
+
+speculos_expected_chaincode = "007c54db71630a77129b2183b701a6da"
+speculos_expected_chaincode += "1cde07a1f4edb1d8ee2f51a14306b4c5"
+
+
+def get_review_instructions(num_screen_skip):
+    instructions = [NavIns(NavInsID.RIGHT_CLICK)] * num_screen_skip
+    instructions.append(NavIns(NavInsID.BOTH_CLICK))
+    return instructions
+
+
+def test_eos_mainmenu_and_setting(client, test_name, navigator):
+    eos = EosClient(client)
+
+    # Get appversion and "data_allowed parameter"
+    rapdu: RAPDU = eos.send_get_app_configuration()
+    assert rapdu.status == STATUS_OK
```

Comment:
Not needed, you can implicitly unwrap "rust style"
`data_allowed, version = eos.send_get_app_configuration().data`

Yes it is somewhat slightly inconsistent with the error assertion but it's so convenient that I suggest you try it ;)